src/ResourceManager/Cdn/Commands.Cdn/help/Confirm-AzureRmCdnEndpointProbeURL.md

```diff
@@ -0,0 +1,77 @@
+---
+external help file: Microsoft.Azure.Commands.Cdn.dll-Help.xml
+Module Name: AzureRM.Cdn
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.cdn/confirm-azurermcdnendpointprobeurl
+schema: 2.0.0
+---
+
+# Confirm-AzureRmCdnEndpointProbeURL
+
+## SYNOPSIS
+Validates a probe URL.
+
+## SYNTAX
+
+```
+Confirm-AzureRmCdnEndpointProbeURL -ProbeUrl <String> [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Confirm-AzureRmCdnEndpointProbeURL** cmdlet confirms if the probe URL provided can be used for dynamic site acceleration.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Confirm-AzureRmCdnEndpointProbeURL -ProbeUrl "http://www.bing.com/images"
```

Comment:
@prakharsharma10 for the cmdlets you are adding in this PR, would you mind updating the examples for their markdown files to include the output of the command that is run? That will give users a better idea of what is returned from the cmdlets

Reply:
@cormacpayne We don't have the examples for many of the cmdlets and as we are working on a release, it's not a high priority for us right now. Some of the customers have already requested for these cmdlets and would appreciate if we could publish these cmdlets ASAP and add the examples later. Thanks!

Reply:
@prakharsharma10 This is a requirement for changes in the stable branch.  Please make this change.